tests/scripts/generate_tls13_compat_tests.py

```diff
@@ -375,6 +375,40 @@ def generate_compat_test(server=None, client=None, cipher=None, sig_alg=None, na
     return '\n'.join(server_object.pre_checks() + client_object.pre_checks() + [cmd])
 
 
+def generate_hrr_compat_test(server=None, client=None, cipher=None, sig_alg=None,
```

Comment:
```suggestion
def generate_hrr_compat_test(client=None, server=None, cert_sig_alg=None,
                                                  client_named_group=None, server_named_group=None):
```
To put things closer to the order they are used, no need for a `cipher` argument and the signature algorithm is to select the certificate thus better to name it `cert_sig_alg`.